cloudant-client/src/main/java/com/cloudant/client/api/query/package-info.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+/**
+ * <p>
+ * This package provides access to the
+ * <a
+ * href="https://console.bluemix.net/docs/services/Cloudant/api/cloudant_query.html"
+ * target="_blank">query API</a>.
+ * </p>
+ * <h1>Overview</h1>
+ * <p>
+ * This package contains classes which make it easier to interact with the query API in the
+ * following ways:
+ * </p>
+ * <ul>
+ *     <li>
+ *         {@link com.cloudant.client.api.query.QueryBuilder A class for building complex queries}
+ *         without having to write them "manually" in JSON.
```

Comment:
`write them "manually" in JSON`
I know what you mean and the use of `"` suggests to me that you were trying to think of a better way to phrase this, fwiw the best I can think of is
>without having to compose the string representation of the query JSON

which isn't a shining exemplar of the English language :)

Reply:
Yes I was struggling for the right expression - how does 034f6ec look?